Player/Player.gd

```diff
@@ -0,0 +1,33 @@
+extends CharacterBody2D
+
+
+const SPEED = 200
+@onready var anim = get_node("AnimatedSprite2D")
```

Comment:
I think this variable should be a constant cause the value never change, and the name could be more self explanatory

Reply:
I'm not 100% sure, but I think nodes can not be `const`

Reply:
Yap, after checking:

> In GDScript, the `const` keyword is used to define constants, and it can only be used to assign simple, immutable values such as numbers and strings. You cannot assign mutable objects like nodes, arrays, dictionaries, or custom objects to a const variable directly.